### DISCUSSION TITLE
NEW text reveal animation

### DISCUSSION THREAD

**Discussion by allen1997831** [elementor/elementor#23303] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Dear Elementor Team,

I am writing to suggest a feature enhancement for your platform. I recently encountered an impressive scroll-triggered text reveal animation on this website: "[](https://markovskaia.ru/en/)". This dynamic effect significantly elevates user engagement and website aesthetics.

It's possible that this effect might be created using the GreenSock Animation Platform (GSAP) or similar. My proposal is to integrate a feature into Elementor to easily add such animations, streamlining this process within your user-friendly interface.

If a full feature integration isn't feasible at the moment, a tutorial or code snippet to guide users on how to implement this effect would be a valuable alternative.

I believe this feature or guidance would enrich Elementor's capabilities and offer users greater design flexibility.

Thanks for considering my suggestion.
Allen

### Describe the solution you'd like

adding any script or built-in new animation in "advanced tab" and motion effects and add this animation.

**Comment by datyorubadev** [comment]
Dear Elementor Team, we will be glad if this is implemented in the future updates, by adding interaction timeline as a feature of Elementor. So, we will be control and create our text animation either by animating the word, line and char. Thank you.

**Comment by cranstantinople** [comment]
It would be great if there was a click to reveal widget or setting on text widgets to change the text on hover/click with CAPTCHA verification.  This would be useful for portfolios to hide personal information such as email and phone numbers from bots.
